### DISCUSSION TITLE
Plot has a farmer public key that is not in the farmer's pk list

### DISCUSSION THREAD

**Discussion by sadegh1010** [Chia-Network/chia-blockchain#8638] [Q&A]
I made  trouble mistake. I reinstall windows on main machine. 
all of backups,farmer key gone 
I have 24 words
more than 600 plots don't work
and now my farmer machine log shows "plot has a farmer public key that is not in the farmer's pk list".
I've added 24 seeds.
i don't have previous farmer public key!
and now I have no idea!

**Comment by BrandtH22** [comment]
@sadegh1010 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
